main.go

```diff
@@ -287,14 +287,12 @@ func main() {
 
 	<-consumer.Ready()
 	var minBlock int
```

Comment:
I'm not sure why this condition was present but it makes testing with a null broker problematic and I can't think of a good reason why it was there in the first place.